<launch>

	<!-- DETECTNET -->
	<arg name="model_name" default="ssd-mobilenet.onnx"/>
	<arg name="model_path" default="/home/jetbot/Desktop/vehicle/datasets/model/ssd-mobilenet.onnx"/>
	<arg name="prototxt_path" default=""/>
	<arg name="class_labels_path" default="/home/jetbot/Desktop/vehicle/datasets/model/labels.txt"/>
	<arg name="input_blob" default="input_0"/>
	<arg name="output_cvg" default="scores"/>
	<arg name="output_bbox" default="boxes"/>
	<arg name="overlay_flags" default="box,labels,conf"/>
	<arg name="mean_pixel_value" default="0.0"/>
	<arg name="threshold" default="0.5"/>

	<node pkg="ros_deep_learning" type="detectnet" name="detectnet" output="screen">
		<remap from="/detectnet/image_in" to="/video_source/compress"/>
		<param name="model_name" value="$(arg model_name)"/>
		<param name="model_path" value="$(arg model_path)"/>
		<param name="prototxt_path" value="$(arg prototxt_path)"/>
		<param name="class_labels_path" value="$(arg class_labels_path)"/>
		<param name="input_blob" value="$(arg input_blob)"/>
		<param name="output_cvg" value="$(arg output_cvg)"/>
		<param name="output_bbox" value="$(arg output_bbox)"/>
		<param name="overlay_flags" value="$(arg overlay_flags)"/>
		<param name="mean_pixel_value" value="$(arg mean_pixel_value)"/>
		<param name="threshold" value="$(arg threshold)"/>
	</node>

</launch>